be/src/runtime/mem_tracker.h

```diff
@@ -97,8 +104,9 @@ class MemTracker {
     // Gets a shared_ptr to the "process" tracker, creating it if necessary.
     static std::shared_ptr<MemTracker> get_process_tracker();
     static MemTracker* get_raw_process_tracker();
-    // Gets a shared_ptr to the "brpc server" tracker, creating it if necessary.
-    static std::shared_ptr<MemTracker> get_brpc_server_tracker();
+    // Get a temporary tracker with a specified label, and the tracker will be created when the label is first get.
+    // Temporary trackers are not automatically destructed, which is usually used for debugging.
+    static std::shared_ptr<MemTracker> get_temporary_mem_tracker(const std::string& label);
```

Comment:
where is `get_brpc_server_tracker`?

Reply:
removed `brpc_server_tracker`,
The test found that the values ​​of `brpc_server_tracker` and `process_tracker` are almost the same, I understand that most of the operations of Doris are `brpc` calls.